README.md

```diff
@@ -42,9 +42,10 @@ Navigate to your Jekyll project directory and create a Guardfile using:
 Or if you already have a Guardfile, add a Jekyll guard.
 
 ```ruby
-guard "jekyll_plus" do
+ignore /^_site/ # NOTE: this can interfere with Guard::LiveReload
```

Comment:
@parkr - ignore is a global setting (it's a Listen setting, not something specific to the watcher or guard). Ideally, source files should be in a subdir, like `src`, because Listen is recursive by default. (you can't ignore _site, because the Guard::LiveReload watchers won't fire).